src/test/java/org/sonarlint/intellij/telemetry/TelemetryClientAttributeProviderImplTest.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * SonarLint for IntelliJ IDEA
+ * Copyright (C) 2015-2021 SonarSource
+ * sonarlint@sonarsource.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonarlint.intellij.telemetry
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.sonarlint.intellij.AbstractSonarLintLightTests
+import org.sonarlint.intellij.config.Settings
+
+class TelemetryClientAttributeProviderImplTest : AbstractSonarLintLightTests() {
+
+    val underTest = TelemetryClientAttributeProviderImpl()
+
+    @Test
+    fun defaultDisabledRules() {
+        assertThat(underTest.defaultDisabledRules).isEmpty()
+
+        Settings.getGlobalSettings().disableRule("ruleKey1")
+        Settings.getGlobalSettings().disableRule("ruleKey1")
+        Settings.getGlobalSettings().disableRule("ruleKey2")
+
+        assertThat(underTest.defaultDisabledRules).hasSize(2)
+        assertThat(underTest.defaultDisabledRules).containsOnly("ruleKey1", "ruleKey2")
```

Comment:
`containsOnly` will already assert the size, so no need for the `hasSize` assertion

Reply:
Same.